python/README.md

```diff
@@ -18,3 +18,20 @@ In order to obtain relevant modules, you should install them from PyPI directly,
 ```
 pip install geoarrow-rust-core
 ```
+
+## Development
+
+To install versions of the package under active development, clone the repo and open the `python` directory:
+
+```
+gh repo clone geoarrow/geoarrow-rs
+cd geoarrow-rs/python
```

Comment:
You need to additionally `cd` into one of the python packages. You can't build anything from the python dir.

Reply:
:+1: